docs/conf/eu/2017/news/announcing-presentations-ticket-update.rst

```diff
@@ -0,0 +1,63 @@
+* Raphael Pierzina - `Hi, my name is README! </conf/eu/2017/speakers/#speaker-raphael-pierzina>`_
+* Ruthie BenDor - `Even Naming This Talk Is Hard </conf/eu/2017/speakers/#speaker-ruthie-bendor>`_
+* Stuart Culshaw - `Tech Writers Without Borders: Making the world a better place, one (numbered) step at a time </conf/eu/2017/speakers/#speaker-stuart-culshaw>`_
+* Thomas Parisot - `Writing a book in 2017  </conf/eu/2017/speakers/#speaker-thomas-parisot>`_
+* Tim Rogers - `"You never get a second chance to make a first impression": writing great "getting started" documentation </conf/eu/2017/speakers/#speaker-tim-rogers>`_
+
+Unconference Sessions
+---------------------
+
+* Ed Stenson - `One-on-One 101 </conf/eu/2017/speakers/#speaker-ed-stenson>`_
+* Floriana Pagano - `Managing community-driven documentation - how to herd doc-writing cats through motivation and a pinch of automation </conf/eu/2017/speakers/#speaker-floriana-pagano>`_
+* Kathleen De Roo - `Manage your docs like an archivist </conf/eu/2017/speakers/#speaker-kathleen-de-roo>`_
+
+Important Ticket Pricing Update
+-------------------------------
+
+**The short version**: Based on community feedback, we have adjusted our ticket pricing to be more accessible and inclusive to our European community, and refunded the difference to those who already purchased tickets.
+You can check out the new ticket pricing on our `Tickets <http://www.writethedocs.org/conf/eu/2017/tickets/>`_ page.
+
+**The long version**: Each year we examine our ticket pricing structure and determine the different tiers based on factors like production costs, attendee experience, and sponsorship fund offset.
+Even though we needed to make adjustments over the years, our goal has always been to maintain a lower price point and therefore a lower entry barrier to our events, so that documentarians from all walks of life can attend.
+
+This year we determined a ticket increase based on many factors, but it turns out that our calculations were simply wrong. There, we said it.
+We received some candid feedback from different members of our community who represent the diverse backgrounds and locations of our attendees, and we listened.
+
+Our events are not-for-profit, and even though we do maintain a small (but highly dedicated!) community staff on retainer, the engagement of the documentarians in our community is what keeps the family growing.
+We are also, alas, human. Mistakes can and will be made, but standing by a mistake is sometimes more damaging than owning up and fixing it. We can make better mistakes tomorrow.
+
+We hope that the new pricing structure better reflects our core values of accessibility and inclusivity. We are grateful to our fellow documentarians who communicated their feedback gracefully and helped us through this process.
+We already processed partial refunds for the price difference for those who already registered, but it might take a few days for the bank or credit card transactions.
```

Comment:
Nowhere in here do we mention that we are lowering prices. It seems really vague and also doesn't really explain where we went wrong. I think we need to rework this.